src/features/click-to-load.js

```diff
@@ -27,9 +28,12 @@ const titleID = 'DuckDuckGoPrivacyEssentialsCTLElementTitle'
 let config = null
 let sharedStrings = null
 let styles = null
-// Used to choose between extension/desktop flow or mobile apps flow.
-// Updated on ClickToLoad.init()
-let isMobileApp
+
+/**
+ * List of platforms where we can skip showing a Web Modal from C-S-S.
+ * It is generally expected that the platform will show a native modal instead.
+ * @type {import('../utils').Platform["name"][]} */
+const platformsToSkipCSSModal = ['android', 'ios']
```

Comment:
Nits:

- Maybe `platformsWithNavitveModalSupport`?
- It would be nice to have the similar arrays (or ideally sets) for mobile platforms and custom element support here too.
- Maybe add the checking method for this one to DDGWidget too, or maybe remove the checking methods for the other ones? I don't mind either way, it just seems kind of inconsistent at the moment.

Reply:
> I can't add this check to a method in the DDGWidget too because there isn't always a widget...

Ah right, got it.

>  I'd like to start avoiding having a lot of config floating in the module scope.

I don't feel too strongly. To me, these are constants that are not really a configuration, but a enumeration of the code's capabilities. In the future if more platforms were added to support native dialogs, the constant could be updated with that change. That said, I don't feel strongly and I'd rather not  bike shed it. So feel free to put those constants where you think makes most sense!